### DISCUSSION TITLE
Blog comment on Cell Theory - Planning for organic growth

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#2011] [Docs/blog comments]
# Blog comment on Cell Theory - Planning for organic growth

The last six months we’ve felt a quantum shift at Weaviate...

https://weaviate.io/company/playbook/cell-theory-planning-for-organic-growth

**Comment by EdwardTSmith42** [comment]
I loved the thoughtfulness of even thinking of this and I wish you the best in your quality cell production 🤘 I loved this:
"That way, cell leaders always have the essential organizational knowledge and can be guardians of the culture."
